### PR TITLE
Improve client-side logging

### PR DESCRIPTION
**Pull request by Crispioso**
### What

Improved some of the logging in Florence to help us when a content loss issue happens:
- Add logging on 'GET content' to help get coverage on what information is coming back from Zebedee after a 'POST content'.
- Log timestamps on post content, get content, disable Florence UI and enable Florence UI.
- Remove ping request results from the console and store in a global variable.

### How to review

1) Switch to the branch `feature/logging`
2) Open a bulletin in the editor and check the GET and POST request for the page content are logged.
3) Pings should no longer show in the console but are available with the `Florence.ping` global object.

### Who can review

Anyone but me.

### CODE REVIEW

File: dist/legacy-assets/js/main.js [hunk 1]
```diff
@@ -295,6 +308,9 @@ function getPageData(collectionId, path, success, error) {
     dataType: 'json',
     type: 'GET',
     success: function (response) {
+      var date = new Date();
+      date = date.getHours() + ":" + date.getMinutes() + "." + date.getMilliseconds();
```
Comment (ian-kent) [inline dist/legacy-assets/js/main.js]: this line is repeated quite a lot - could be worth creating a log function which does it for you?

File: dist/legacy-assets/js/main.js [hunk 2]
```diff
@@ -11539,34 +11560,27 @@ function setShortcuts(field, callback) {
...
 
-                if (averagePingTime < 100) {
-                    console.log("ping time: pretty good! " + time + " average: " + averagePingTime + " " + pingTimes);
-                } else if (averagePingTime < 300) {
-                    console.log("ping time: not so good! " + time + " average: " + averagePingTime + " " + pingTimes);
-                } else {
-                    console.log("ping time: really bad! " + time);
-                }
+                var date = new Date();
+                Florence.ping.add(time, date)
```
Comment (ian-kent) [inline dist/legacy-assets/js/main.js]: should `add` need a time?

Comment (Crispioso) [inline dist/legacy-assets/js/main.js]: Good point, resolved 👍

File: dist/legacy-assets/js/main.js (hunk 1 above)
Comment (Crispioso) [inline dist/legacy-assets/js/main.js]: Perhaps worth doing if we use it more often? We haven't got any global logging function yet but could be worth adding it into something like that if we find ourselves putting more time into it.

**Pull request merged**